FacuReartes/react-laba-final-project-team-2
# SPT2 184 My Wishlist Page

## Changes

- Create My Wishlist Page UI & functionalities.

### Jira

- [SPT2-184](https://solvd-final-project-team-2.atlassian.net/jira/software/projects/SPT2/boards/3?selectedIssue=SPT2-184)

### How to Test

1. Add a product to the wishlist for example in the home page, then go to profile/wishlist and you can see the product is added.

### Screenshots (if applicable)
![image](https://github.com/user-attachments/assets/65b7bd08-d99e-4757-80b8-665f08f9f100)
![image](https://github.com/user-attachments/assets/d88e5749-03d9-42ef-baa9-1f847b6fdaff)
![image](https://github.com/user-attachments/assets/e2eb4c83-b8ee-46fe-86b9-2ea139f8792f)

### Checklist


[SPT2-184]: https://solvd-final-project-team-2.atlassian.net/browse/SPT2-184?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
